StereoKitC/platforms/linux.cpp

```diff
@@ -518,9 +509,10 @@ void linux_shutdown() {
 ///////////////////////////////////////////
 
 void linux_step_begin_xr() {
-  #if !defined(SKG_LINUX_EGL)
-	linux_events();
+  #if defined(SKG_LINUX_EGL)
```

Comment:
Not my project and not my style guide but I really feel like you should expand this to `#if` cases and use brackets

Reply:
@maluoi thoughts?

Reply:
This looks kinda gnarly, I'm inclined to agree that this needs re-worked.